www-misc/zoneminder/zoneminder-1.34.8.ebuild

```diff
@@ -0,0 +1,178 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# TO DO:
+# * dependencies of unknown status:
+#	dev-perl/Device-SerialPort
+#	dev-perl/MIME-Lite
+#	dev-perl/MIME-tools
+#	dev-perl/PHP-Serialization
+#	virtual/perl-Archive-Tar
+#	virtual/perl-libnet
+#	virtual/perl-Module-Load
+
+EAPI=6
+
+inherit eapi7-ver perl-functions readme.gentoo-r1 cmake-utils depend.apache flag-o-matic systemd
```

Comment:
Please sort this as much as possible (you may move eclasses around based on their `EXPORT_FUNCTIONS` values)

Reply:
Is there a script for that?

Reply:
Not that I know of, of course you can make a simple loop that `cd`'s to `portagedir/eclass` then runs a simple grep given of input list of eclasses.

`EXPORT_FUNCTIONS` is something you learn by writing ebuilds, truthfully I wouldn't know if all of these eclasses use it or not either (because I am not familiar with `perl-functions` or `depend.apache`, but they don't seem like eclasses to define default phases)

Reply:
sorted them, it still emerges, so probably no inter-eclass dependencies (if any) are broken